instrumentation/apache-dubbo-2.7/library-autoconfigure/src/main/java/io/opentelemetry/instrumentation/apachedubbo/v2_7/DubboTelemetry.java

```diff
@@ -13,9 +13,18 @@
 /** Entrypoint for instrumenting Apache Dubbo servers and clients. */
 public final class DubboTelemetry {
 
-  /** Returns a new {@link DubboTelemetry} configured with the given {@link OpenTelemetry}. */
-  public static DubboTelemetry create(OpenTelemetry openTelemetry) {
-    return builder(openTelemetry).build();
+  /**
+   * Returns a new client {@link DubboTelemetry} configured with the given {@link OpenTelemetry}.
+   */
+  public static DubboTelemetry createClient(OpenTelemetry openTelemetry) {
+    return builder(openTelemetry).buildClient();
+  }
+
+  /**
+   * Returns a new server {@link DubboTelemetry} configured with the given {@link OpenTelemetry}.
+   */
+  public static DubboTelemetry createServer(OpenTelemetry openTelemetry) {
+    return builder(openTelemetry).buildServer();
   }
```

Comment:
This does not align with existing instrumentations that instrument both client and server, such as https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/grpc-1.6/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6, where `TelemetryBuilder` builds a single `Telemetry` that can be used to produce either client or server interceptor.

Reply:
> This does not align with existing instrumentations that instrument both client and server, such as https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/grpc-1.6/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6, where `TelemetryBuilder` builds a single `Telemetry` that can be used to produce either client or server interceptor.

Thanks, I modified it, Please have a look again!